### DISCUSSION TITLE
Some text searches failing (on private repo)

### DISCUSSION THREAD

**Discussion by mbaas2** [community/community#11594] [Other Feature Feedback, Questions, & Ideas] [answered]
I've searched `noprint` in a private repo.  It wasn't found - even though there is at least one .html file in a subfolder that contains this term.

(Maybe my emphasis on "private repo" is wrong, but this is the only thing special I could think of - as I saw cs working satisfyingly on other repos before)
But according to #10526 I think it is safe to expect private repos to be indexed as well - so my case was on https://github.com/Dyalog/contest-content/  (and `noprint`can be found in https://github.com/Dyalog/contest-content/blob/staging/assets/welcome.html right at the top)

**Comment by look** [comment, marked as answer]
Hi @mbaas2, I can't check on this because it's a private repository, but we do have some heuristics that exclude some content, notably files we detect as "vendored" or "generated" (you can [read more in our FAQ](https://cs.github.com/about/faq#indexed-content)). That could be why this file is missing.

However, we're working on changing this behavior because it's often counterproductive.
